bazel/repositories.bzl

```diff
@@ -156,6 +156,10 @@ def envoy_dependencies(skip_targets = []):
     _com_lightstep_tracer_cpp()
     _io_opentracing_cpp()
     _net_zlib()
+    _repository_impl("com_github_gflags_gflags")
```

Comment:
I don't think this is used?

Reply:
Used transitively in a test. Needs to be cleaned up, just like the use of glog. Waiting for absl::log to rule them all.

Reply:
You're not running tests from those dependencies so you don't have to have this, no? I couldn't find where this used

Reply:
glog uses it https://github.com/google/glog/blob/master/WORKSPACE#L4

Reply:
Yes I see that, the point is as long as you don't depends on any rules depends on `@ com_github_gflags_gflags//blah`, you don't need it in workspace definition. Do we use it really?

Reply:
If i remove it, envoy doesn't build... Maybe bazel is now more restrictive or something.

Reply:
hmm ok seems glog requires gflags...

Reply:
I'll remove both eventually in cel-cpp. The delayed release of absl components makes this harder than necessary.